kbase-extension/static/kbase/js/widgets/appWidgets2/input/dynamicDropdownInput.js

```diff
@@ -272,7 +300,7 @@ define([
                         // could check here that each item is a map? YAGNI
                         obj = flattenObject(obj);
                         if (!('id' in obj)) {
-                            obj.id = _index; // what the fuck
+                            obj.id = _index;
```

Comment:
Every object passed to Select2 needs a unique `id` field.

Reply:
if only the original author of that line had read the docs!